src/App.js

```diff
@@ -1,16 +1,48 @@
 import React from "react";
 import logo from "./logo.png";
+import Clock from "./Clock.js";
+import { Container, Row, Col } from "react-bootstrap";
 import "./App.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+function MyLayout() {
```

Comment:
I think we could name this a bit better. This is very vague and achieves the same as "App" in regards to component naming